final-project/manJoinProcess.tex

```diff
@@ -63,4 +63,16 @@ \subsection{Messaging Protocol}
 
 \section{Creating or Loading Rooms}
 
+The client will have three options: generating a temporary room with a uuid, creating a temporary room with inputs, and creating a permanent room that can be rejoined in the future.
```

Comment:
add a note that creating a temporary room with parameters is not possible from the web UI, it's merely a feature supported by the REST API.